tests/OutputFormatTest.php

```diff
@@ -11,7 +11,7 @@
 use Sabberworm\CSS\Parsing\OutputException;
 
 /**
- * @covers \Sabberworm\CSS\OutputFormat
+ * @coversNothing
```

Comment:
Isn't this covering various `OutputFormat` methods (`create*` and some of the setters)?

Reply:
As far as I understand this test, it's not about covering how `OutputFormat` works, but about how other elements get rendered depending on the output format.